pybv/io.py

```diff
@@ -74,9 +75,13 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
         resolution, and writing data in µV with 0.1 resolution will thus
         guarantee accurate writing vor all values >= 1e-7 µV
         (``1e-6 * 0.1``).
-    unit : str
+    unit : str | list of str
```

Comment:
the text under the `data` parameter needs to be modified as well, because also non-Volt data is now "allowed" as well, but the interplay is a bit complex with the `unit` param, so we need to brainstorm for a good phrasing.

Potentially it'd be good to also add a short example to the README or the docstring (if feasible in only a few lines)